### PR TITLE
Update kubelet overview help doc

### PR DESCRIPTION
**Pull request by deitch**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation

> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

Completes documentation change updated as discussed with @liggitt at https://github.com/kubernetes/website/pull/12336

**Special notes for your reviewer**:
Minor documentation change. Originally was placed in https://github.com/kubernetes/website, but that page is auto-generated.

**Does this PR introduce a user-facing change?**:

```release-note
Update kubelet CLI summary documentation and generated Webpage
```

cc @liggitt

### CODE REVIEW

**Comment by liggitt** [comment]
/ok-to-test
/lgtm

will defer to kubelet approvers
/assign @yujuhong

**Comment by deitch** [comment]
Can anyone explain why this failed? All I changed is a bit of text.

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -118,7 +118,10 @@ func NewKubeletCommand(stopCh <-chan struct{}) *cobra.Command {
 	cmd := &cobra.Command{
 		Use: componentKubelet,
 		Long: `The kubelet is the primary "node agent" that runs on each
-node. The kubelet works in terms of a PodSpec. A PodSpec is a YAML or JSON object
+node. It registers the node with the apiserver using one of: the hostname; a flag to
```
[the remark this answers is not in the document]
Comment (deitch) [reply cmd/kubelet/app/server.go]: Very good point. Updating now.

[the remark this answers is not in the document]
Comment (deitch) [reply cmd/kubelet/app/server.go]: Done.

**Pull request merged**